Node Status Monitor/NodeJS/logs/error/error.log

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
the output was under folder "error" , since we cannot upload blank folder therefore we kept empty file inside it.
if the folder doesn't exist it may create issue in execution